.github/workflows/test_on_push.yml

```diff
@@ -132,7 +131,6 @@ jobs:
 
   # Skips IDAKLU module compilation for speedups, which is already tested in other jobs.
   run_doctests:
-    needs: style
```

Comment:
If style fails then PR is updated with the pre-commit fixes and the old jobs get canceled. This just speeds up the start of the jobs a bit